test/framework/src/main/java/org/elasticsearch/script/MockScriptEngine.java

```diff
@@ -739,6 +739,11 @@ public boolean needs_score() {
                     return true;
                 }
 
+                @Override
+                public boolean needs_termStats() {
+                    return false;
```

Comment:
I prefer to keep false since we would have to implement a mock for ScriptTermStats in that case.
BTW, there is specific tests for the termStats use case.